docs/source/antennas_iq.rst

```diff
@@ -152,11 +171,12 @@ The file fields in the antennas_iq array files are:
 +-----------------------------------+---------------------------------------------+
 | | **pulse_phase_offset**          | | For pulse encoding phase, in degrees      |
 | | *float32*                       | | offset. Contains one phase offset per     | 
-| | [number of pulses]              | | pulse in pulses.                          |
+| | [] or [num pulses x 1] or       | | pulse in pulses or one phase offset       |
```

Comment:
Not sure exactly how this is used (like how the dimensions work). Need to review it further